deployments/ansible/roles/collector/README.md

```diff
@@ -34,6 +34,13 @@ how to use the role in a playbook with minimal required configuration:
         splunk_realm: SPLUNK_REALM
 ```
 
+You can disable starting the collector, smart agent, and fluentd services by setting 
+the argument `start_service` to `false`:
+
+```
+$> ansible-playbook playbook.yaml -e start_service=false
```

Comment:
Should `start_service` also be available as a role variable?

Reply:
yes ok, let's do that.